dev-util/snowman/snowman-0.1.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A native code to C/C++ decompiler"
+DESCRIPTION="Snowman is a native code to C/C++ decompiler, \
+supporting x86, AMD64, and ARM architectures. You can use it \
+as a standalone GUI application, command-line tool, IDA plug-in \
+or a library. Snowman is free software.\
+* Supports ARM, x86, and x86-64 architectures.
+* Reads ELF, Mach-O, and PE file formats
+* Reconstructs functions, their names and arguments, local and global variables,
+expressions, integer, pointer and structural types, all types of control-flow
+structures, including switch.
+* Has a nice graphical user interface with one-click navigation between the
+assembler code and the reconstructed program.
+* Has a command-line interface for batch processing.
+"
+HOMEPAGE="https://derevenets.com"
+COPYRIGHT="2010 Yegor Derevenets"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yegord/snowman/archive/v$portVersion.zip"
+CHECKSUM_SHA256="45a9bf4dd0805c1c42155b6c5ff1c86bb28273dbf1453e958cfeee3cec086bc0"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	snowman$secondaryArchSuffix
+	app:snowman$secondaryArchSuffix
```

Comment:
@diversys @pulkomandy  I've ammended the recipe with the suggested changes, are there any other issues with it?